modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -799,14 +799,20 @@ else if (navigation.equals("mine") || navigation.equals("recent")) {
 	private Hits _getHits(SearchContainer<RepositoryEntry> searchContainer)
 		throws PortalException {
 
+		long searchRepositoryId = ParamUtil.getLong(
+			_httpServletRequest, "searchRepositoryId");
+
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)_httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		themeDisplay.setScopeGroupId(searchRepositoryId);
```

Comment:
Before do this we should check that 
`if (searchRepositoryId != 0) {`

Reply:
[modules-integration-mysql57-jdk8/0/8](https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/381456//consoleText)
[com.liferay.document.library.web.internal.display.context.test.DLAdminDisplayContextTest](https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/381456/testReport/com.liferay.document.library.web.internal.display.context.test/DLAdminDisplayContextTest) - [Failed 0 of last 25](https://testray.liferay.com/home/-/testray/case_results/2195006105/history)

fails for this reason :)

Reply:
I fixed it. thank @AliciaGarciaGarcia